src/eastwood/linters/implicit_dependencies.clj

```diff
@@ -5,8 +5,7 @@
 
 
 (defn var->ns-symbol [var]
-  (let [^clojure.lang.Namespace ns (-> var meta :ns)]
-    (.-name ns)))
+  (symbol (namespace (symbol var))))
```

Comment:
Probably this bugfix is important enough to deserve a test

Reply:
Yes. I'm open for suggestions on how to use/mimic `#'potemkin/import-vars`. I'm not sure how to replicate without introducing new (test) dependencies.

Reply:
> I looked how #'potemkin/import-vars works, and added a test which fails without the bugfix.

Checked out, LGTM! Nice to test out these vars without pulling a .jar dep